Crop_image_main.py

```diff
@@ -10,42 +10,115 @@
 python crop_main.py <image> <metadatafile> <output_crop>
```

Comment:
It's not related to changes here but this comment is no longer accurate for the script name.
```suggestion
Crop_image_main.py image metadatafile output_crop
```

Reply:
agree